tests/scripts/payment/mark_payments_as_sent_test.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import patch
+
+from pcapi.core.payments import factories as payment_factories
+from pcapi.models.payment_status import PaymentStatus
+from pcapi.models.payment_status import TransactionStatus
+from pcapi.scripts.payment.mark_payments_as_sent import mark_payments_as_sent
+
+
+class MarkPaymentsAsSentTest:
+    def test_mark_payments_as_sent(self, app, db_session):
+        # Given
+        payment1 = payment_factories.PaymentFactory(id=100, statuses=[], transactionLabel="transaction_label")
+        payment2 = payment_factories.PaymentFactory(id=101, statuses=[], transactionLabel="transaction_label")
+        payment3 = payment_factories.PaymentFactory(id=102, statuses=[], transactionLabel="transaction_label")
+        payment4 = payment_factories.PaymentFactory(id=103, statuses=[], transactionLabel="transaction_label")
+        no_status_under_review_payment = payment_factories.PaymentFactory(
+            id=104, statuses=[], transactionLabel="transaction_label"
+        )
+        wrong_transaction_label_payment = payment_factories.PaymentFactory(
+            id=105, statuses=[], transactionLabel="wrong_transaction_label"
+        )
+        payments_to_be_marked_as_sent = [payment1, payment2, payment3, payment4]
```

Comment:
```suggestion
        expected_payments_to_be_marked_as_sent = [payment1, payment2, payment3, payment4]
```

Un peu plus long, certes, mais c'est plus explicite et ça rend le code plus bas un peu plus clair quand on compare ce qu'on a (en base de données) et ce à quoi on s'attend.